bundler.d/development.rb

```diff
@@ -7,12 +7,14 @@
   # for generating foreign key migrations
   gem 'immigrant', '~> 0.1'
 
+  # debugging and REPL tools
+  gem 'byebug'
   gem 'pry'
   gem 'pry-rails'
   gem 'pry-byebug'
   gem 'pry-doc'
   gem 'pry-stack_explorer'
-  gem 'pry-remote'
+  gem 'pry-remote' # will not work when running via foreman
```

Comment:
I think, that in that case we should remove it from here, as it has not a lot of value here then.

Reply:
I thought so but @ares still uses it according to the discourse discussion. This won't hurt I think.

Reply:
This was recently introduced (merged by me) in believe MOST developers are using it with `bundle exec foreman`, but if it's used just by few developers out of nostalgia, we should not have it here, they can add it to the `Gemfile.local` as they did prior merge. This should IMHO include only gems used by most developers and gems we want to promote to new developers as "the best way"

Reply:
Do not start with the best way in software engineering please, I say tabs are better than spaces. :-)

This is not a big deal, if you do not like it I am going to close the PR as I plan to work on better script that I described in the thread that will actually let us start foreman with our without the foreman gem (a single tool for both) so pre-remote will work again.

Reply:
Nah, I like adding byebug, I was just trying to tag more changes along, but I'm ok to merge as-is :+1: